src/app/components/UpcomingLivestreamCard.js

```diff
@@ -3,22 +3,45 @@
 import { useEffect, useState } from "react";
 
 export const UpcomingLivestreamCard = ({ video, ...props }) => {
-  const [availableAt, setAvailableAt] = useState("");
+  const [streamDay, setStreamDay] = useState(0);
+  const [streamMonth, setStreamMonth] = useState(0);
+  const [streamYear, setStreamYear] = useState(0);
+  const [streamTime, setStreamTime] = useState("");
+
+  var monthNames = ["JAN", "FEB", "MAR", "APR", "MAY","JUNE","JULY", "AUG", "SEP", "OCT", "NOV","DEC"];
 
   useEffect(() => {
     const date = new Date(video.available_at);
-    setAvailableAt(date.toLocaleString());
-    console.log(date.toLocaleString());
+    setStreamDay(date.getDay());
```

Comment:
`date.getDay()` will output the day (monday 0, tuesday 1, etc). I think what you meant to do is `date.getDate()`
```suggestion
    setStreamDay(date.getDate());
```

Reply:
oo u ryt, thats basically what i meant lol.

Reply:
yeah just feel free to look at the changes and take what you need